features/step_definitions/project_steps.rb

```diff
@@ -43,12 +43,8 @@
   end
 end
 
-When("the project is approved") do
-  app.project_under_test.approve
-end
-
-When("the project is rejected") do
-  app.project_under_test.reject
+When(/the project receives (approval|rejection)/) do |status_change_factory|
```

Comment:
This feels like DRY for DRY's sake.

Reply:
It's DRY because I had to change it several times.